cookbook/guarded.ts

```diff
@@ -0,0 +1,128 @@
+import path from "path"; // md-ignore
+import { Account, DevnetEntrypoint } from "../src"; // md-ignore
+// md-start
+(async () => {
+    // ### Guarded transactions
+    // Similar to relayers, transactions also have two additional fields:
+
+    // - guardian
+    // - guardianSignature
+
+    // Each controller includes an argument for the guardian. The transaction can either:
+    // 1. Be sent to a service that signs it using the guardian’s account, or
+    // 2. Be signed by another account acting as a guardian.
+
+    // Let’s issue a token using a guarded account:
+
+    // #### Creating guarded transactions using controllers
+    // We can create guarded transactions using any of the available controllers.
+
+    // Each controller method includes a guardian argument, which must be set if we want to create a guarded transaction.
+    // Let’s issue a fungible token using a relayed transaction:
```

Comment:
For a separate PR - maybe we change to do simpler transactions (simple native transfers) to exemplify the usage of guardians in controllers? If we decide to do this, we should update the Python cookbook, as well.